### PR TITLE
Update standard tests

### PR DESCRIPTION
**Pull request by awirb**
* Add Warning in report if min difference exceeds threshold 
* add standardTests cfg file

### CODE REVIEW

File: avaframe/com1DFA/standardTests_com1DFACfg.ini [hunk 1]
```diff
@@ -0,0 +1,48 @@
+### Config File - This file contains the main settings for the simulation run
+## Copy to  local_com1DFACfg.ini and set you parameters
+# This file is part of Avaframe.
+
+
+[GENERAL]
+# Path to samos executable and AK_Attributes files
+samosAT = /home/anna/repos/gitea/AvaFrame_SamosDFA/samosAT -files /home/anna/repos/gitea/AvaFrame_SamosDFA/files/AK_Attributes/
```
Comment (fso42) [inline avaframe/com1DFA/standardTests_com1DFACfg.ini:8]: Please remove your local path :-)

File: avaframe/log2Report/generateCompareReportCfg.ini [hunk 2]
```diff
@@ -2,6 +2,9 @@
 ## Copy to  local_outQuickPlotCfg.ini and set you parameters
 # This file is part of Avaframe.
 
+[GENERAL]
+# limit to produce warning for difference in peak fields
+diffLim = 0.00001
```
Comment (fso42) [inline avaframe/log2Report/generateCompareReportCfg.ini:7]: Should we maybe use percentage instead of a fixed value?

**Review by fso42 — CHANGES_REQUESTED** [review]
I like your idea of having a seperate .ini file. I just wonder if we could take it a step further and dump the current benchmark ini into each of our test cases and load it for every case seperately?

File: avaframe/com1DFA/standardTests_com1DFACfg.ini (hunk 1 above)
Comment (awirb) [reply avaframe/com1DFA/standardTests_com1DFACfg.ini:8]: Done

File: avaframe/log2Report/generateCompareReportCfg.ini (hunk 2 above)
Comment (awirb) [reply avaframe/log2Report/generateCompareReportCfg.ini:7]: Good idea! What percentage of the min difference would be suitable?

Comment (fso42) [reply avaframe/log2Report/generateCompareReportCfg.ini:7]: Mh, that's up for discussion. How about we start with 0.01 % ?

Comment (awirb) [reply avaframe/log2Report/generateCompareReportCfg.ini]: fine!

**Comment by awirb** [comment]
* now a avaName_com1DFACfg.ini file for each benchmark located in the benchmark folders
* samosAT executable path still read form com1DFACfg.ini
* Warning if difference is diffLim percent of max value of benchmark result

**Review by fso42 — APPROVED** [review]

**Pull request merged**